### PR TITLE
github: add template text as comments

### PR DESCRIPTION
**Pull request by nashif**
Instead of including the boilerplate text and showing it in the created
issue, use comments that are still displayed when editing, but not shown
in the final issue, avoiding confusing and mixup between actual issue
and template text.

Signed-off-by: Anas Nashif <anas.nashif@intel.com>

### CODE REVIEW

**Review by fabiobaltieri — DISMISSED** [review]
Can't wait to see how many issues we'll get with the description hidden. Ship it!

File: .github/ISSUE_TEMPLATE/001_bug_report.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ labels: bug
 assignees: ''
 
 ---
-
+<!--
 **Notes (delete this)**
```
Comment (pdgendt) [inline .github/ISSUE_TEMPLATE/001_bug_report.md:10]: This line can be removed IMO.

**Review by pdgendt — DISMISSED** [review]

**Review by aescolar — DISMISSED** [review]
Thank you :)

**Review by aescolar — DISMISSED** [review]

**Review by fabiobaltieri — DISMISSED** [review]

**Review by pdgendt — DISMISSED** [review]

File: .github/ISSUE_TEMPLATE/001_bug_report.md [hunk 2]
```diff
@@ -27,31 +29,44 @@ the problem you're facing:
...
 copy-and-paste text and put a code fence (\`\`\`) before and after, to help
 explain the issue. (if unable to obtain text log, add a screenshot)
+-->
 
 **Environment (please complete the following information):**
+<!--
  - OS: (e.g. Linux, MacOS, Windows)
  - Toolchain (e.g Zephyr SDK, ...)
  - Commit SHA or Version used
+-->
```
Comment (henrikbrixandersen) [inline .github/ISSUE_TEMPLATE/001_bug_report.md:66]: Shouldn't this be left as-is for people to fill in?

File: .github/ISSUE_TEMPLATE/001_bug_report.md [hunk 3]
```diff
@@ -27,31 +29,44 @@ the problem you're facing:
...
 
 **To Reproduce**
+<!--
 Steps to reproduce the behavior:
 1. mkdir build; cd build
 2. cmake -DBOARD=board\_xyz
 3. make
 4. See error
+-->
```
Comment (henrikbrixandersen) [inline .github/ISSUE_TEMPLATE/001_bug_report.md:41]: Shouldn't this be left as-is for people to fill in?

**Review by henrikbrixandersen — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/001_bug_report.md (hunk 3 above)
Comment (nashif) [reply .github/ISSUE_TEMPLATE/001_bug_report.md:41]: that, I am not sure we should leave, the text above is not fill-in really, there are different ways steps can added, the text serves as example.

Comment (henrikbrixandersen) [reply .github/ISSUE_TEMPLATE/001_bug_report.md:41]: That's fair, but can we at least leave the SHA, platform, and toolchain visible?

File: .github/ISSUE_TEMPLATE/001_bug_report.md (hunk 2 above)
Comment (nashif) [reply .github/ISSUE_TEMPLATE/001_bug_report.md]: yeah, left as is

File: .github/ISSUE_TEMPLATE/001_bug_report.md (hunk 3 above)
Comment (nashif) [reply .github/ISSUE_TEMPLATE/001_bug_report.md:41]: yes, I did that already, forgot to post comment...

**Review by pdgendt — APPROVED** [review]

**Review by henrikbrixandersen — APPROVED** [review]

**Pull request merged**